HHS/OPRE-OPS
# Ops 992 notification api

## What changed

Added initial Notification Center.

## Issue

https://app.zenhub.com/workspaces/opre-ops-62d197b4c468970013bcd2ec/issues/gh/hhs/opre-ops/992

## How to test

1. Log into the application and click the bell icon next to your email.
2. Add a Notification object to the DB
3. The notification should appear in the UI within 5 s
